receiver/frequency_far_jump/frequency_far_jump.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Plugin: Jump to a frequency outside the boundaries of the selected profile by typing it in the receiver's frequency dial.
+ * Requires the option 'Allow users to change center frequency' to be enabled in the admin panel.
+ *
```

Comment:
Also, please add the "beware" note here too. So it will be available to all who want to enable the plugin.